### PR TITLE
Post cantina

### PR DESCRIPTION
**Pull request by Rubilmax**
Contains:
- #51 
  - Missing interface for `ChainlinkOracle`
- #55
  - Add Natspecs to interface

### CODE REVIEW

**Review by Rubilmax — COMMENTED** [review]
Missing the morpho-blue submodule checkout update

File: src/interfaces/IChainlinkOracle.sol [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.5.0;
+
+import {IERC4626} from "./IERC4626.sol";
+import {AggregatorV3Interface} from "./AggregatorV3Interface.sol";
+import {IOracle} from "../../lib/morpho-blue/src/interfaces/IOracle.sol";
+
+interface IChainlinkOracle is IOracle {
```
Comment (StErMi) [inline src/interfaces/IChainlinkOracle.sol:8]: Consider following the same best practice you have already used in other interface contracts by covering also the interface with full natspec doc tags. 

Consider adding at least @title and @notice (to describe the interface)

Comment (StErMi) [reply src/interfaces/IChainlinkOracle.sol:8]: To be clear, I mean something along the line of

```solidity
/// @title IChainlinkOracle
/// @author Morpho Labs
/// @custom:contact security@morpho.org
```

**Comment by StErMi** [comment]
Looks good to me, I've left some info comments where needed.

File: src/interfaces/IChainlinkOracle.sol (hunk 1 above)
Comment (MerlinEgalite) [reply src/interfaces/IChainlinkOracle.sol:12]: Fixed here #55

**Pull request merged**